src/main/java/bio/terra/service/job/JobMapKeys.java

```diff
@@ -22,7 +22,8 @@ public enum JobMapKeys {
   FILE_ID("fileId"),
   ASSET_ID("assetId"),
   TRANSACTION_ID("transactionId"),
-  DELETE_CLOUD_RESOURCES("deleteCloudResources");
+  DELETE_CLOUD_RESOURCES("deleteCloudResources"),
+  SOURCE_DATASET("sourceDataset");
```

Comment:
I was expecting this to be an ID
```suggestion
  SOURCE_DATASET_ID("sourceDatasetId");
```

Actually it looks like `DATASET_ID` is used below so I think this can be reverted.